pandas/core/frame.py

```diff
@@ -2654,16 +2654,16 @@ def memory_usage(self, index=True, deep=False) -> Series:
         Examples
         --------
         >>> dtypes = ['int64', 'float64', 'complex128', 'object', 'bool']
-        >>> data = dict([(t, np.ones(shape=5000).astype(t))
+        >>> data = dict([(t, np.ones(shape=5000, dtype=int).astype(t))
```

Comment:
Had to change because the previous output was due to the buggy display of object dtyped floats with a zero fractional part as ints (same as L2791):
```python
[ins] In [4]: arr
Out[4]: array([1.0, 2.0], dtype=object)

[ins] In [5]: pd.Series(arr)
Out[5]: 
0    1
1    2
dtype: object
```